hd/lang/lex_utf8.txt

```diff
@@ -41,6 +41,18 @@ fr: de
 en: officer/officer/officers
 fr: officiant/officiante/officiants
 
+    name changed
+en: the first_name, surname or occurence of this eprson has changed. Please update linked pages accordingly
+fr: le prénom, nom ou occurence de cette personne a changé. Pensez à mettre à jour les pages liées associées    
```

Comment:
À relire, je me dis que si possible, ce serait mieux de faire une seule entrée au lexique en simplifiant un peu l'avertissement dont le texte est légèrement redondant :
« Le prénom, nom ou occurrence de cette personne a changé. [S’il en existe] Pensez à mettre à jour les %s à l’ancienne clé. » avec le lien HTML « pages liées » sur le %s.
“The first name, name or occurrence of this person has changed. Please update %s to the old key.”

Peut-être nom/name plutôt que clé/key, c'était juste pour éviter la confusion des mots « nom » avec deux sens différents mais clé n'est pas très sympatique…

En passant j'avais raté qu'il manquait deux r aux « occur[r]ences », faute fréquente ! ^^